client/session.go

```diff
@@ -66,9 +66,21 @@ func (x ResSessionCreate) PublicKey() []byte {
 // see [apistatus] package for NeoFS-specific error types.
 //
 // Context is required and must not be nil. It is used for network communication.
+//
+// Return errors:
+//   - [ErrMissingSigner]
 func (c *Client) SessionCreate(ctx context.Context, prm PrmSessionCreate) (*ResSessionCreate, error) {
+	signer := prm.signer
+	if signer == nil {
```

Comment:
may be shrinked
```go
if prm.signer == nil {
  prm.signer = c.prm.signer
  if prm.signer == nil {
    return nil, ErrMissingSigner
  }
}
```

Reply:
it is one line shorter but an if-in-if nesting that i do not like usually. also, your way does some implicit changes (the default signer becomes the signer from `prm` ) which i do not like too